tests/test-javalin-jsonb/src/main/java/org/example/myapp/web/GetBeanForm.java

```diff
@@ -30,6 +32,12 @@ public class GetBeanForm {
 
   @QueryParam private Set<ServerType> type;
 
+  @Ignore @IgnoreBeanField private String ignored;
```

Comment:
I'm not 100% convinced on this / `@IgnoreBeanField`. My gut says the field should be `transient` and that jsonb and avaje-http should honour transient fields as being ignored.

Reply:
you know, I always forget that is a valid identifier

Reply:
I see that transient is used for serialization only though

Reply:
> I see that transient is used for serialization only though

The transient modifier isn't _just_ used by Java serialisation.  JPA / ORMs similarly honour transient fields/properties and avaje-jsonb _should_ honour transient fields in that jsonb should treat transient fields as `@Json.Ignore` by default.

My thought is that "beans" with transient fields should by default be ignored in terms of query parameters or form parameters.

Reply:
I meant it's used to say a field should not be serialized, can't seem to find where it says that it also means a field cannot be deserialized.